<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>1161</width>
     <height>631</height>
    </rect>
   </property>
   <widget class="QWidget" name="page_calculate">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_trangle"/>
  </widget>
  <widget class="QPushButton" name="btn_calculate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>普通计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_trangle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>三角形相关</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
